l10n_ch_dta/wizard/create_dta.py

```diff
@@ -684,7 +684,7 @@ def _process_payment_lines(self, data, pline, elec_context, seq):
         number = number.replace('.', '').replace('-', '') or False
         elec_context['partner_bank_number'] = number
         elec_context['partner_bvr'] = ''
-        if pline.partner_bank_id.acc_type in ('bv', 'bvr', 'postal'):
+        if pline.partner_bank_id.acc_type == 'postal':
```

Comment:
Second thinking about it:

This is not what we want.

Here it won't work with a postal number as IBAN or the CCP of the bank of your bank account.

As we can have a

- Postal account with a ccp number
- Postal account with an iban number
- Bank account for which the bank has a ccp number

In all those 3 case `ccp` will be filled with the ccp

```
- a postal account, we use acc_number
- a postal account in iban format, we transform acc_number
- a bank account with CCP on the bank, we use ccp of the bank
- otherwise there is no CCP to use
```

What we probably want to test here is if `pline.partner_bank_id.ccp` is set.